packages/ui/internal/Pure/tests/Pure-specs.js

```diff
@@ -0,0 +1,77 @@
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+import {Component} from 'react';
+
+import Pure from '../Pure';
+
+const defaultConfig = {
+	propKeys: [1],
+	nextKeys: [1],
+	propComparators: {
+		'*': (a, b) => a === b
+	},
+	hasChanged: jest.fn()
+};
+
+let data = [];
+
+class Base extends Component {
+	constructor (props) {
+		super(props);
+	}
+
+	render () {
+		data = this.props;
+		return <div {...this.props} data-testid="baseComponent">Value: {this.props.a}</div>;
```

Comment:
how about replacing "a" with "value" ?
It should also be destructured because you pass {..this.props} as attributes for the div, as well as "this.props.a" as child